### PR TITLE
Es6 projects common modules commercial donot use adblock

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

ES6. Flow.

## What is the value of this and can you measure success?

Safer, clearer, newer

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        INT: {
+            campaign: 'ADB_INT',
+            messageText: [
+                "We notice you're using an ad-blocker. Perhaps you'll support us another way?",
+                'Become a Supporter for less than $1/€1 per week',
+            ].join(' '),
+            linkText: 'Find out more',
+        },
+    };
+    const message = messages[config.page.edition];
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js]:
```js
config.get('page.edition')
```

File: static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                messageText: message.messageText,
+                linkText: message.linkText,
+                arrowWhiteRight: inlineSvg('arrowWhiteRight'),
+            })
+        );
+    }
+};
+
+const showAdblockBanner = (): void => {
+    const banners = getBanners(config.page.edition);
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js]:
```js
config.get('page.edition')
```

File: static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    const flatBanners = [];
+    banners.forEach(bannerList => {
+        flatBanners.push(sample(bannerList));
+    });
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js]:
```js
flatbanners = banners.map(/*...*/)
```
?

**Review by GHaberis — APPROVED** [review]
Nice 👍

**Pull request merged**